tests/x11regressions/brasero/brasero_launch.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2017 SUSE LLC
```

Comment:
It is new. so only
`Copyright © 2017 SUSE LLC`

Reply:
fixed